extensions/adobe/experience/customerJourneyManagement/push-channel-context.example.1.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Mobile team should also review if they haven't already

Reply:
There is an ambiguity on Mobile team right now. We are waiting for some discussion to finalize and will then float it to them